spec/PlaygroundNet46TestSpec.js

```diff
@@ -0,0 +1,30 @@
+var selenium = require('selenium-webdriver')
```

Comment:
Very simple jasmine test spec that will start Playbround.Net46 and attempt to find a control with a testID = "WelcomeText"